lib/src/collect.dart

```diff
@@ -271,22 +306,12 @@ Future<void> _processFunction(VmService service, IsolateRef isolateRef,
       }
       return;
     }
+    final hits = getHitMap(Uri.parse(script.uri!));
+    hits.funcHits ??= <int, int>{};
+    hits.funcNames ??= <int, String>{};
```

Comment:
nit. You *could* merge these lines.

```suggestion
    (hits.funcNames ??= <int, String>{})[line] = funcName;
```

🤷  – get rid of a null assert.